src/proteomes/components/results/__tests__/__snapshots__/ProteomesCard.spec.tsx.snap

```diff
@@ -0,0 +1,146 @@
+                  Homo sapiens (Human)
+                </a>
+              </span>
+              <span
+                class="result-card__info-bit"
+              >
+                Protein Count: 
+                <a
+                  href="/uniprotkb?query=proteome:UP000005640"
+                >
+                  18
+                </a>
+              </span>
+              <span
+                class="result-card__info-bit"
+              >
+                Genome representation: full
+              </span>
+              <span
+                class="result-card__info-bit"
+              >
+                <abbr
+                  title="Complete Proteome Detector"
+                >
+                  CPD
+                </abbr>
+                : Standard
+              </span>
+            </section>
+            <section>
+              <span
+                class="result-card__info-bit"
+              >
+                <abbr
+                  title="Benchmarking Universal Single-Copy Orthologs"
+                >
+                  BUSCO
+                </abbr>
+                <br />
+                <span>
+                  <span
+                    class="busco-legend-item"
+                  >
+                    <span
+                      class="busco-legend-item__swatch"
+                      style="background-color: rgb(188, 232, 117);"
+                    />
+                    Single
+                  </span>
+                  <span
+                    class="busco-legend-item"
+                  >
+                    <span
+                      class="busco-legend-item__swatch"
+                      style="background-color: rgb(11, 165, 166);"
+                    />
+                    Duplicated
+                  </span>
+                  <span
+                    class="busco-legend-item"
+                  >
+                    <span
+                      class="busco-legend-item__swatch"
+                      style="background-color: rgb(238, 203, 164);"
+                    />
+                    Fragmented
+                  </span>
+                  <span
+                    class="busco-legend-item"
+                  >
+                    <span
+                      class="busco-legend-item__swatch"
+                      style="background-color: rgb(49, 75, 180);"
+                    />
+                    Missing
+                  </span>
+                </span>
+                : 
+                <div
+                  class="busco-view"
+                >
+                  <protvista-track
+                    height="10"
+                    length="100"
+                  />
+                  <div>
+                    n:103 · lineageDb value
+                  </div>
+                  <div>
+                    C:77.7% (S:7.8% D:11.7%) F:17.5% M:19.4%
+                  </div>
+                </div>
+              </span>
+            </section>
```

Comment:
There are 4 levels of sections within one single card entity. It defeats a bit the whole idea of defining semantic sections.
At most I would accept the card top-level to be a section, anything below it not really

Reply:
I think this is a bit of a generic problem for the codebase in general but I can update it for Cards for a start (other cards also suffer from this issue). I'm working on the PR branching off this PR at the moment, would you mind if I addressed it there?